src/client/java/minicraft/level/Chunk.java

```diff
@@ -0,0 +1,39 @@
+package minicraft.level;
+
+import minicraft.level.tile.Tile;
+import minicraft.level.tile.Tiles;
+
+public class Chunk {
+	public static final int SIZE = 32;
```

Comment:
Chunks should probably be much larger. Perhaps 64?

Reply:
Agreed, I was hoping to get feedback on this. Even 128 would be fine, but 64 is probably a good size. A 64x64 chunk would probably cover the whole screen which I think is a good metric, may be useful for potential chunk tricks like caching things or something like that. Not sure what intended use for chunks are being considered.